test/CMakeLists.txt

```diff
@@ -16,6 +16,6 @@ list(APPEND CMAKE_MODULE_PATH ${catch2_SOURCE_DIR}/contrib)
 add_executable(testrunner
                TestDefs.cc TestRunner.cc
                TestPreferenceList.cc TestPartnerList.cc TestUtils.cc
-               TestStableMarriage.cc TestPopular.cc TestHR.cc TestYokoi.cc TestMaximalEnvyfreeHRLQ.cc TestParser.cc)
+               TestStableMarriage.cc TestPopular.cc TestHR.cc TestYokoi.cc TestMaximalEnvyfreeHRLQ.cc TestParser.cc TestFavouriteNeighbour.cc TestPrefers.cc)
```

Comment:
TestFavouriteNeighbour.cc and TestPrefers.cc contains code that should be part of the same file, maybe call that TestTies.cc.